package.json

```diff
@@ -154,13 +155,14 @@
     "husky": "7.0.4",
     "jest": "27.4.5",
     "jest-watch-typeahead": "1.0.0",
-    "lerna": "4.0.0",
+    "lerna": "^6.0.1",
```

Comment:
Can we set the exact version, without "^"?
And does it actually just works when we jump from v4 to v6?=)

Reply:
We did not use it to its full potential, so there should be no problems, everything is fine locally